packages/block-editor/src/components/rich-text/index.js

```diff
@@ -111,7 +111,7 @@ export function RichTextWrapper(
 		__unstableDisableFormats: disableFormats,
 		disableLineBreaks,
 		__unstableAllowPrefixTransformations,
-		disableEditing,
+		readonly,
```

Comment:
Should it be `readOnly` instead? `O` uppercase.

Reply:
Yes, good catch https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#readonly